nuxeo-features/nuxeo-platform-rendition/nuxeo-platform-rendition-core/src/test/java/org/nuxeo/ecm/platform/rendition/service/TestRenditionOperation.java

```diff
@@ -82,6 +83,7 @@
     protected RenditionService renditionService;
 
     @Test
+    @ConditionalIgnoreRule.Ignore(condition = ConditionalIgnoreRule.IgnoreWindows.class, cause = "NXP-26757")
```

Comment:
Wdyt about putting it on the class since all tests have it (except one which expect a `NuxeoException`) ?